src/Common.Projects.props

```diff
@@ -1,7 +1,7 @@
 <Project>
     <PropertyGroup>
         <Company>Trakx</Company>
-        <Copyright>Copyright © 2021 Trakx</Copyright>
+        <Copyright>Copyright © 2022 Trakx</Copyright>
```

Comment:
Not a blocker for now, but now it is 2023.

Reply:
thanks I opened #113 for it :)